src/test/ui/union/union-const-eval.rs

```diff
@@ -1,8 +1,4 @@
-// build-pass (FIXME(62277): could be check-pass?)
-// revisions: mirunsafeck thirunsafeck
-// [thirunsafeck]compile-flags: -Z thir-unsafeck
```

Comment:
This seems strange, why remove the THIR unsafeck part of the test?

Reply:
Probably an oversight when I saw the "could be check-pass" comment. Is there documentation somewhere for the headers? I've seen quite a few things but never actually seen an explanation of them.

Reply:
There's some documentation at https://rustc-dev-guide.rust-lang.org/tests/adding.html. If there's something missing, that's probably worth a bug report (or PR :D ) at https://github.com/rust-lang/rustc-dev-guide/.